modules/python/test/test_feature_homography.py

```diff
@@ -42,6 +43,7 @@ class feature_homography_test(NewOpenCVTests):
 
     def test_feature_homography(self):
 
+        random.seed(0)
```

Comment:
`TestSceneRender` class uses `cv2.randn` random generator from OpenCV. Your patch does not affect the test behavior at all.